h/schemas/api/group.py

```diff
@@ -91,6 +92,7 @@ def _validate_groupid(self, appstruct):
             # The authority part of the ``groupid`` doesn't match the
             # group's authority
             raise ValidationError(
+                # pylint:disable=consider-using-f-string
                 "{err_msg} '{groupid}'".format(
                     err_msg=_("Invalid authority specified in groupid"), groupid=groupid
                 )
```

Comment:
Yep. Again I think mixing the `_(` into the string directly would lead to syntax nightmares. I mean I come from a Perl background, but still...